cpt/runner.py

```diff
@@ -125,14 +131,21 @@ def run(self):
                                                         not_export=self.skip_recipe_export,
                                                         update=self._update_dependencies)
                             else:
+                                if self._profile_build_abs_path is not None:
+                                    profile_build = ProfileData([self._profile_build_abs_path],
```

Comment:
`ProfileData` has 5 fields but only 4 are set
https://github.com/conan-io/conan/blob/addd8ab4726a557d165e17ec6bb202f0c1666f0c/conans/client/conan_api.py#L74

Reply:
This branch has been implemented using v1.35 of Conan
https://github.com/conan-io/conan/blob/release/1.35/conans/client/conan_api.py#L74

In that version only 4 values were required.
I'll look into that today.